app/helpers/application_helper.rb

```diff
@@ -642,4 +642,31 @@ def list_to_string(list)
   end
 
 
+  def t(*args)
+    translation = I18n.t(*args).downcase
+    term =  "ontology"
+    plural_term = "ontologies"
+    single_term = "an ontology"
```

Comment:
this will not work in languages other than English.

Reply:
no it works

Reply:
fixed